src/modules/supply.ts

```diff
@@ -1,14 +1,51 @@
-import axios, { AxiosInstance } from 'axios';
 import { getConfig } from '../bootstrap';
 import BlockchainModule from './base';
 
+interface SupplyBaseResponse {
+  /** Current block in the chain */
+  height: string;
+}
+
+export interface SupplyResult {
+  /** The name of the coin */
+  denom: string;
+  /** Total supply in string format */
+  amount: string;
+}
+
+export interface TotalSupplyResponse extends SupplyBaseResponse {
+  /** Array of totals supplies  */
+  result: SupplyResult[];
+}
+
+export interface TotalSupplyDenomResponse extends SupplyBaseResponse {
+  /** Amount of total supply for specific denom */
+  result: string;
+}
+
+export interface SupplyError {
```

Comment:
Isn't this very common between modules?

Reply:
Yeap, it is. @jsandovalg and I decided to create our own types and after get all modules done refactor common types to avoid comflicts between commits

Reply:
Ok makes sense. @jsandovalg @David-Sod create a new ticket to reflect that refactor specifically so we don't forget.

Reply:
Comment the ticket here and I'll merge this.

Reply:
Sure, this issue was created at #63